frontend/src/metabase/query_builder/components/expressions/Editor/Shortcuts/Shortcuts.tsx

```diff
@@ -0,0 +1,45 @@
+import cx from "classnames";
+
+import { Button, Flex, Icon, type IconName } from "metabase/ui";
+
+import S from "./Shortcuts.module.css";
+
+export type Shortcut = {
+  name: string;
+  icon: IconName;
+  action: () => void;
+};
+
+const DEFAULT_SHORTCUTS: Shortcut[] = [];
+
+export function Shortcuts({
+  shortcuts = DEFAULT_SHORTCUTS,
+  hide = false,
+}: {
+  shortcuts?: Shortcut[];
+  hide?: boolean;
+}) {
+  return (
+    <Flex
+      gap="sm"
+      className={cx(S.shortcuts, {
+        [S.hidden]: hide,
+      })}
+    >
+      {shortcuts.map((shortcut, index) => (
+        <Button
+          key={index}
+          variant="light"
+          radius="5rem"
+          size="xs"
+          onClick={shortcut.action}
```

Comment:
The shortcuts do not work for me, they just disappear on click.

Reply:
Not the case for me, in all three browsers I tested. Is this still the case for you, or have I accidentally fixed the issue?